<launch>

    <!-- set parameters and custom transforms -->
    <include file="$(dirname)/frames.launch"/>

    <param name="/move_group/planning_plugin" value="pilz_industrial_motion_planner::CommandPlanner"/>

    <!-- start simulation -->
    <include file="$(find thi_franka_moveit_config)/launch/demo.launch" pass_all_args="true">
        <arg name="rviz_tutorial"        value="true"/>
        <arg name="pipeline"             value="pilz_industrial_motion_planner"/>
    </include>

</launch>